yarn.lock

```diff
@@ -5681,7 +5681,7 @@ execall@^1.0.0:
   dependencies:
     clone-regexp "^1.0.0"
 
-exenv@^1.2.0, exenv@^1.2.1:
+exenv@^1.2.0, exenv@^1.2.1, exenv@^1.2.2:
```

Comment:
Though this doesn't seem to belong in this pr 🤔

Reply:
Hmm, yarn decided to do that and I decided to `git commit -a`... whoops

Reply:
Thanks for the review @AllenBW, just rebased and removed this yarn.lock change.